src/ol_superset/requirements.txt

```diff
@@ -0,0 +1,468 @@
+#
```

Comment:
We can't use pyproject.toml instead?

Reply:
Not really, because we need to generate a requirements file that is included as part of the image build for Superset. I started by including it as part of pyproject.toml, but it caused downgrades of packages that we rely on for other things. This seemed like the best option to have the requirements that we need for Superset, while also having a lock-file, and being able to optionally install them in a local venv for purposes of auto-complete during development.